PickUp89/PickUpBackend
# Create Google Auth Endpoint

**How to test:**
- http://localhost:13000/auth/google 
- currently only support test account, contact Jay to get your google account added
- If success, will be direct to http://localhost:13000/success
- Use DBeaver to check if new user has been added to the DB
